### DISCUSSION TITLE
Start Icon not changing to Windows 11 and also not centering.

### DISCUSSION THREAD

**Discussion by ruZeph** [valinet/ExplorerPatcher#2288] [Q&A]
As mentioned in the title, I've enabled these settings:
![image](https://github.com/valinet/ExplorerPatcher/assets/92640971/a766d234-5a5e-47fd-813a-7ba70c2580d7)

But my start icon is not changing to Windows 11 type and not centering as you can see below.
![image](https://github.com/valinet/ExplorerPatcher/assets/92640971/fdaf8e04-7caf-41f6-a5b3-398fc2bd1075)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
